### PR TITLE
fixing tree view

### PR DESCRIPTION
**Pull request by olhabn**
image

nanopay pr: https://github.com/nanoPayinc/NANOPAY/pull/11880

### CODE REVIEW

File: src/foam/nanos/menu/VerticalMenu.js [hunk 1]
```diff
@@ -114,7 +105,7 @@ foam.CLASS({
               startExpanded: true,
               query: self.menuSearch$,
               onClickAddOn: function(data) { self.openMenu(data); },
-              selection$: self.currentMenu$,
+              selection: self.currentMenu,
```
Comment (olhabn) [inline src/foam/nanos/menu/VerticalMenu.js:108]: it creates a bug for menus under 'Data Managment'

Comment (olhabn) [reply src/foam/nanos/menu/VerticalMenu.js:108]: and it's not a good fix but I don't have a better solution

File: src/foam/u2/view/TreeView.js [hunk 2]
```diff
@@ -200,37 +207,44 @@ foam.CLASS({
...
               show(this.hasChildren$).
               style({
-                'visibility': 'visible',
-                'font-size':  '16px',
-                'transform':  this.expanded$.map(function(c) { return c ? 'rotate(180deg)' : 'rotate(90deg)'; })
+                'visibility':    'visible',
+                'font-size':     '16px',
+                'float':         'right',
+                'height':        '6px',
+                'padding':       self.expanded$.map(function(c) { return c ? '5px 0px 8px 3px' : '0px 0px 11px 4px'}),
```
Comment (olhabn) [inline src/foam/u2/view/TreeView.js]: I don't have a better solution for making `toggle-icon` vertically aligned

File: src/foam/u2/view/TableView.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ foam.CLASS({
   extends: 'foam.u2.view.UnstyledTableView',
 
   css: `
+    ^ {
+      overflow-x: scroll;
```
Comment (kgrgreer) [inline src/foam/u2/view/TableView.js]: Would this be better if it were 'auto'?

File: src/foam/u2/view/TreeView.js [hunk 4]
```diff
@@ -200,37 +207,44 @@ foam.CLASS({
...
-              }
-            }, self.level$, this.selection$, this.data$.dot('id'))).
-            start().
-              addClass(self.myClass('select-level')).
-              style({
-                'width': '100%',
-                'padding-right': '20px'
-              }).
-              addClass(self.myClass('label')).
+              else
```
Comment (kgrgreer) [inline src/foam/u2/view/TreeView.js]: No need for 'else'.

File: src/foam/u2/view/TreeView.js (hunk 2 above)
Comment (kgrgreer) [reply src/foam/u2/view/TreeView.js]: That's fine. CSS isn't an elegant tool.

Comment (nanopay-arthur) [reply src/foam/u2/view/TreeView.js]: What if we make toggle icon's width equal to its height? And if it's still off we can just add some padding to the bottom?

Comment (olhabn) [reply src/foam/u2/view/TreeView.js]: thanks!

File: src/foam/u2/view/TableView.js (hunk 3 above)
Comment (olhabn) [reply src/foam/u2/view/TableView.js]: fixed

File: src/foam/u2/view/TreeView.js (hunk 4 above)
Comment (olhabn) [reply src/foam/u2/view/TreeView.js]: fixed

**Pull request merged**